### PR TITLE
Wordpress 5.5.1 added

### PR DESCRIPTION
**Pull request by javicv**
Hello

I think Wordpress is a great addition to the marketplace in order to allow people to create his own web/blog inside the cluster.

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

I tried to install the _app.yaml_ with _kubectl_ and it works fine, as you can see in the next screenshot:
wordpress_k3s_civo

The only doubt I have with this application is, as long as it requires MariaDB, how would the marketplace install MariaDB without a plan selection from the user?

### CODE REVIEW

**Comment by saiyam1814** [comment]
@javicv Let's put it under management category for now, once we have something similar coming up again we will thing of putting a new category.
Can you add the category in the manifest file ?

**Comment by javicv** [comment]
> @javicv Let's put it under management category for now, once we have something similar coming up again we will think of putting a new category.
> Can you add the category in the manifest file ?

Of course @saiyam1814, changed and commited. :wink:

File: wordpress/app.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
```
Comment (saiyam1814) [inline wordpress/app.yaml:2]: we need pv as well right to bound it to pvc ?

Comment (javicv) [reply wordpress/app.yaml:2]: I don't think so. When the PVC is created Longhorn will create the PV under the hood and the PVC is bounded to it.
image

Comment (saiyam1814) [reply wordpress/app.yaml:2]: image
I was trying out on a new cluster with Longhorn installed and I see this

Comment (saiyam1814) [reply wordpress/app.yaml:2]: ok my cluster had some problems, so it was having issues creating the Pv's. This looks good. Will merge

Comment (javicv) [reply wordpress/app.yaml:2]: I'm glad to read that, not your problems with the cluster, but the merge. :wink:

Comment (saiyam1814) [reply wordpress/app.yaml:2]: in few minutes it would be live I would want you to test it on a fresh new cluster so that we can announce that ;)

**Comment by saiyam1814** [comment]
@javicv can you ping me on civo slack to discuss about the post install ?

**Pull request merged**